icons/command.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g fill="none" stroke="#42a5f5" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><path d="M21 22.5H3A1.5 1.5 0 011.5 21V3A1.5 1.5 0 013 1.5h18A1.5 1.5 0 0122.5 3v18a1.5 1.5 0 01-1.5 1.5z"/><path d="M16.917 7.083c.777.776.777 2.036 0 2.813s-3.395.582-3.395.582-.195-2.619.582-3.395 2.037-.777 2.813 0zM7.083 7.083c-.777.776-.777 2.036 0 2.813s3.395.582 3.395.582.195-2.619-.582-3.395-2.037-.777-2.813 0zM16.917 16.917c.777-.776.777-2.036 0-2.813s-3.395-.582-3.395-.582-.195 2.619.582 3.395 2.037.777 2.813 0zM7.083 16.917c-.777-.776-.777-2.036 0-2.813s3.395-.582 3.395-.582.195 2.619-.582 3.395-2.037.777-2.813 0zM10.5 10.5h3v3h-3z"/></g></svg>
```

Comment:
Please remove the surrounding border for this icons and increase the size of the command icon.